torchvision/prototype/datasets/_builtin/pcam.py

```diff
@@ -116,3 +128,7 @@ def _make_datapipe(
         dp = hint_shuffling(dp)
         dp = hint_sharding(dp)
         return Mapper(dp, self._prepare_sample)
+
+    def __len__(self):
+        # TODO imlement len
+        return 3
```

Comment:
Just marking so we don't forget. Do you have the files locally? Otherwise I can provide you the number of samples.